### ISSUE TITLE
Layout/MultilineBlockLayout cop auto-correct ignoring LineLength

### ISSUE THREAD

**Issue by aried3r** [rubocop-hq/rubocop#5788]
When having a lambda block with many parameters, the `Layout/MultilineBlockLayout` cop ignores the maximum configured line length.

Might be related to https://github.com/bbatsov/rubocop/issues/3359

--------

## Expected behavior

Rubocop should respect the configured `LineLength`. Not sure what the right output might look like.

## Actual behavior

```Ruby
some_result = lambda do |so_many, parameters, it_will, be_too_long, for_one_line, but_the_cop, does_not_care, so_much|
  do_something
end
```

## Steps to reproduce the problem

```YAML
# .rubocop.yml

Metrics/LineLength:
  Max: 80
```

```ruby
# rubocop_multilineblock.rb

some_result = lambda do |
    so_many,
    parameters,
    it_will,
    be_too_long,
    for_one_line,
    but_the_cop,
    does_not_care,
    so_much|
  do_something
end
```

`rubocop --only Layout/MultilineBlockLayout rubocop_multilineblock.rb`

## RuboCop version

```
$ rubocop -V
0.55.0 (using Parser 2.5.1.0, running on ruby 2.5.1 x86_64-darwin16)
```

**Comment by Drenmi** [comment]
Autocorrect should be a no-op if the corrected source violates `Metrics/LineLength`.

**Comment by AlexWayfer** [comment]
Nice: unfixed bugs become closed due to unfixing. For presentation like "we have not too much open issues".

**Comment by bbatsov** [comment]
@AlexWayfer See https://metaredux.com/posts/2019/05/09/always-be-closing.html

**Comment by jonas054** [comment]
I'm working on a solution.
